package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "seed": "ts-node prisma/seed.ts",
     "start": "nest start",
```

Comment:
**praise**: Including `@nestjs/config` is a good practice for managing environment variables.